dapp/src/pages/DashboardPage/PositionDetails.tsx

```diff
@@ -91,24 +93,39 @@ export default function PositionDetails() {
 
       <HStack w="full" justify="start" flexWrap="wrap" spacing={5}>
         <UserDataSkeleton>
-          <Button
-            {...buttonStyles}
-            onClick={openDepositModal}
-            isDisabled={featureFlags.TVL_ENABLED && tvl.isCapExceeded}
+          <ArrivingSoonTooltip
+            label="This option is not available on mobile yet. Please use the desktop app to deposit."
+            shouldDisplayTooltip={isMobileMode}
           >
-            Deposit
-          </Button>
+            <Button
+              {...buttonStyles}
+              onClick={openDepositModal}
+              isDisabled={
+                (featureFlags.TVL_ENABLED && tvl.isCapExceeded) || isMobileMode
+              }
+            >
+              Deposit
+            </Button>
+          </ArrivingSoonTooltip>
         </UserDataSkeleton>
         {isConnected && activitiesCount > 0 && (
-          <UserDataSkeleton ml={-3}>
+          <UserDataSkeleton>
             <ArrivingSoonTooltip
-              shouldDisplayTooltip={!isWithdrawalFlowEnabled}
+              label={
+                isWithdrawalFlowEnabled && isMobileMode
+                  ? "This option is not available on mobile yet. Please use the desktop app to withdraw."
+                  : undefined
```

Comment:
When the withdrawals flag is disabled, we shouldn't display the "Arriving soon" label, which is the default of the `ArrivingSoonTooltip` component. We will disable the flag only temporarily if there are problems with withdrawal flow.

How about we set:
```suggestion
                isMobileMode
                  ? "This option is not available on mobile yet. Please use the desktop app to withdraw."
                  : "This option is currently not available."
```

In another task, we will need to consider the `ArrivingSoonTooltip` component and how it's used across the dApp. It feels outdated since the features it was protecting have already arrived.

Reply:
https://github.com/thesis/acre/pull/882/commits/e94d20f2dca462c9ad918510809e59875a753842